packages/e2e/src/init.ts

```diff
@@ -40,9 +40,9 @@ const LogLevelSchema = z.enum(['silent', 'info', 'debug']);
 type LogLevel = z.infer<typeof LogLevelSchema>;
 
 // Configurations
-const LIVE_NETWORK_FUNDING_AMOUNT = '0.01';
+const LIVE_NETWORK_FUNDING_AMOUNT = '0.2';
 const LOCAL_NETWORK_FUNDING_AMOUNT = '1';
-const LIVE_NETWORK_LEDGER_DEPOSIT_AMOUNT = '1';
+const LIVE_NETWORK_LEDGER_DEPOSIT_AMOUNT = '1.2';
```

Comment:
Bumped because of failing ci E2E tests running out of funds, could be reverted after @glitch003 reduces fees on naga-test